front/pages/w/[wId]/builder/data-sources/new-public-url.tsx

```diff
@@ -292,6 +306,37 @@ export default function DataSourceNew({
                 size="sm"
                 className="text-sm"
               />
+              <div style={{ display: "none" }}>
```

Comment:
display:none because we do not support subsequent crawling in the backend yet, but will do in the next PR.